modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -53,3 +53,21 @@ function stanford_profile_styles_preprocess_paragraph(&$vars) {
     $vars['#attached']['library'][] = "stanford_profile_styles/stanford_paragraph_wysiwyg";
   }
 }
+
+/**
+ * Implements hook_ckeditor_css_alter().
+ */
+function stanford_profile_styles_ckeditor_css_alter(array &$css, Editor $editor) {
+  if (!$editor->hasAssociatedFilterFormat()) {
+    return;
+  }
+
+  $known_formats = [
+    'stanford_html',
+    'stanford_minimal_html',
+  ];
+
+  if (in_array($editor->getFilterFormat()->id(), $known_formats)) {
```

Comment:
Is this necessary? the profile knows about what filters are available. :man_shrugging:

Reply:
The config is all there so, yes...

Is it necessary? No, but I don't want these styles showing up on wysiwyg editors that we aren't using to display the changed output.

Reply:
but the output styles could be different than what is visible in the ckeditor.

Reply:
How so?

Reply:
if the css is added to the pages but not to the wysiwyg, it produces different "what you see" results

Reply:
Yes, but where is that happening?

Reply:
Stanford Basic is still adding to the ckeditor stylesheet: https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/stanford_basic.info.yml#L34

These specific heading and font style overrides are only for when we are outputting them through wysiwyg content. The original styles should be the same outside of that scope.

Reply:
its not happening right now but it could in the future or if extending this profile. I'm not really on board with the logic but its not a blocker to me for this work.

Reply:
I don't see how this implementation would change anything about future styles not getting into the CKEditor. If you add new stylesheets within other modules or profiles you will have to add the stylesheet to CKEditor manually as well. Nothing gets in automagically. I am very concerned about long term maintenance. Could you outline a scenario where the new code would be expected to be in the WYSIWYG but isn't?